gh-publish.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
```

Comment:
I'm embarrassed to say how long it took me to understand that this script only packages our extension to be upload as an artifact. I think the file name and the use of "publish" in the script are misleading. I thought we're publishing the extension somewhere.